figures/metrics.py

```diff
@@ -362,6 +389,28 @@ def calc_from_user_model():
     return calc_from_user_model()
 
 
+def get_total_site_learners_joined_for_time_period(site, start_date, end_date):
+    """
+    Returns the number of new learners for the time period
+    """
+
+    def calc_from_user_model():
```

Comment:
`calculate_from_user_model` and add a docstring as well

Reply:
@rehan99000 this naming convention is used by figured in other stats of sections, to keep it consistent I have used it.